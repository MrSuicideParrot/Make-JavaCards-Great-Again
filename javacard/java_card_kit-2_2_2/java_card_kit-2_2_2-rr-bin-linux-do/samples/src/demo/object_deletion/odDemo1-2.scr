
//+
// Copyright 2005 Sun Microsystems, Inc. All rights reserved.
// SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
//-

//+
// Workfile:@(#)Header.scr	1.9
// Version:1.9
// Date:01/04/06
//-

powerup;

// Select the installer applet
0x00 0xA4 0x04 0x00 0x09 0xa0 0x00 0x00 0x00 0x62 0x03 0x01 0x08 0x01 0x7F;
// 90 00 = SW_NO_ERROR

// Turn output off during CAP file download
echo "CAP file download section. Output suppressed.";

output off;

//Card was reset

// resume printing output during functional test
output on;

echo "Functional test section.";

//select Applet A's instance
0x00 0xA4 0x04 0x00 0x0B 0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x07 0x01 0x01 0x7F;
// 90 00 SW_NO_ERROR

//request Object Deletion
0x80 0x10 0x00 0x00 0x00 0x7F;
// 90 00 SW_NO_ERROR

//verify reset mem gone
0x80 0x14 0x00 0x00 0x00 0x7F;
// 90 00 SW_NO_ERROR

//set all attributes (including transient arrays) to null. This also requests GC
0x80 0x15 0x00 0x00 0x00 0x7F;
// 90 00 SW_NO_ERROR


powerdown;

