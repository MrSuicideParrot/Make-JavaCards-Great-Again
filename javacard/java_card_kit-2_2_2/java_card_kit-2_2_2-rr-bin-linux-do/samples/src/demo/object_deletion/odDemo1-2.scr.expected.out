CLA: 00, INS: a4, P1: 04, P2: 00, Lc: 09, a0, 00, 00, 00, 62, 03, 01, 08, 01, Le: 00, SW1: 90, SW2: 00
CAP file download section. Output suppressed.
OUTPUT OFF;
OUTPUT ON;
Functional test section.
CLA: 00, INS: a4, P1: 04, P2: 00, Lc: 0b, a0, 00, 00, 00, 62, 03, 01, 0c, 07, 01, 01, Le: 00, SW1: 90, SW2: 00
CLA: 80, INS: 10, P1: 00, P2: 00, Lc: 00, Le: 00, SW1: 90, SW2: 00
CLA: 80, INS: 14, P1: 00, P2: 00, Lc: 00, Le: 00, SW1: 90, SW2: 00
CLA: 80, INS: 15, P1: 00, P2: 00, Lc: 00, Le: 00, SW1: 90, SW2: 00
